tests/unittests/BackendTestUtils.cpp

```diff
@@ -188,10 +188,22 @@ void compareAgainstInterpreter(llvm::StringRef backendName,
   for (Tensor *T : resultTensors) {
     EXPECT_TRUE(IFT.second->isEqual(*T, allowedError));
   }
+
+  // Additionally check that each of the results from the parallel cloned
+  // Functions are bitwise equal.
+  Tensor *firstResult = nullptr;
+  for (Tensor *T : resultTensors) {
+    if (!firstResult) {
```

Comment:
Nit: I think this could be more readable if you move setting firstResult outside the loop.
`Tensor *firstResult = *resultTensors.begin()`